src/pages/Main.tsx

```diff
@@ -1,7 +1,13 @@
 import { FC } from "react";
-import { Button } from "@deskpro/app-sdk";
+import { Button, useInitialisedDeskproAppClient } from "@deskpro/app-sdk";
+import { getShopInfo } from '../context/StoreProvider/api';
 
 export const Main: FC = () => {
+    useInitialisedDeskproAppClient((client) => {
```

Comment:
@zpawn you can remove this block as it's only "temporary" -- used to confirm that we can make API requests

Reply:
@AshleyDawson removed